android/build.gradle

```diff
@@ -49,6 +66,20 @@ android {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
     }
+    if (isNewArchitectureEnabled()) {
+        externalNativeBuild {
+            ndkBuild {
+                path "src/main/jni/Android.mk"
+            }
+        }
+    }
+    packagingOptions {
+        // For some reason gradle only complains about the duplicated versionf of libreact_render libraries
+        // while there are more libraries copied in intermediates folder of the lib build directory, we exlude
+        // only the ones that make the build fail (ideally we should only include librnscreens_modules but we
+        // are only allowed to specify exlude patterns)
+        exclude "**/libreact_render*.so"
+    }
     sourceSets.main {
         java {
             if(isNewArchitectureEnabled()){
```

Comment:
```suggestion
            if (isNewArchitectureEnabled()) {
```

Reply:
Fixed in [7dc787f](https://github.com/software-mansion/react-native-screens/pull/1322/commits/7dc787f2a68e833147d3ca1f05b544d848e6c5ed)